src/containers/EntitySelectBoxContainer.js

```diff
@@ -8,27 +8,36 @@ const propTypes = {
   loadOptions    : PropTypes.func,
   onChange       : PropTypes.func,
   optionRenderer : PropTypes.func,
-  getOption      : PropTypes.func
+  getOption      : PropTypes.func,
+  getGnScores    : PropTypes.func
 }
 
 /**
  * This is the dumb presentation component for the Entity select box.
  */
 class EntitySelectBox extends Component {
   render () {
-    const { loadOptions, onChange, optionRenderer } = this.props
-    return <ReactSelect.Async cache={null}
-      filterOptions={false}
-      loadOptions={loadOptions}
-      onChange={onChange}
-      optionRenderer={optionRenderer} />
+    const { loadOptions, onChange, optionRenderer, getGnScores } = this.props
```

Comment:
This one shouldn't end up in the generic EntitySelectBox I think, cause that makes it unusable elsewhere.